checkstyle.xml

```diff
@@ -34,7 +34,9 @@
 
     <module name="SuppressWarningsFilter"/>
 
-    <!--<module name="NewlineAtEndOfFile" />--> <!-- Screwed up on Appveyor for some reason -->
+    <module name="NewlineAtEndOfFile">
+        <property name="lineSeparator" value="lf"/>
```

Comment:
I don't know. I don't understand the whole Windows line ending thing vs linux. Someone else is going to have to explain/approve this change.
@bradamiller You understand this?